lib/Elevate/Components/AutoSSL.pm

```diff
@@ -16,9 +26,30 @@ use Cpanel::SSL::Auto ();
 
 use parent qw{Elevate::Components::Base};
 
+use Log::Log4perl qw(:easy);
+
+sub check ($self) {
+
+    return $self->_check_autossl_provider();
+}
+
+sub _check_autossl_provider ($self) {
+
+    if ( $self->is_using_sectigo() ) {
+        WARN( <<~"EOS" );
+        Elevating with Sectigo as the provider for AutoSSL is not supported.
+        If you proceed with this upgrade, we will switch your system
+        to use the Let's Encrypt�~D� provider.
```

Comment:
UTF-8 mojibake

Since we pretty much force the `C` locale, maybe replace the screwed up ® or ™ with `(R)` or `(TM)`?